src/compass/utils/helpers.py

```diff
@@ -0,0 +1,108 @@
+'''collection of useful functions used across workflows'''
+
+import os
+
+import journal
+from osgeo import gdal
+
+WORKFLOW_SCRIPTS_DIR = os.path.dirname(os.path.realpath(__file__))
+
+
+def check_file_path(file_path: str) -> None:
+    """Check if file_path and polarizations exist else raise an error.
+
+    Parameters
+    ----------
+    file_path : str
+        Path to file to be checked
+    polarizations : list[str]
+        Path to file to be checked
+    """
+    error_channel = journal.error('helpers.check_file_path')
+    if not os.path.isfile(file_path):
+        err_str = f'{file_path} not found'
+        error_channel.log(err_str)
+        raise FileNotFoundError(err_str)
+
+
+def get_file_mode(file_path: str):
```

Comment:
We can probably be a bit more clear on this function name. when I saw get_file_mode I thought the function checks if a given file is in read or write mode. But all it does is extracting polarization from filename. How about:
```get_polarization_from_filname``` or ```get_polarization```